.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: main
+on: [push, pull_request]
+jobs:
+  test:
+    strategy:
+      matrix:
+        node: ["10.x", "12.x", "14.x"]
+        platform: [ubuntu-latest]
+    name: Node v${{matrix.node}} ((${{matrix.platform}}))
+    runs-on: ${{matrix.platform}}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{matrix.node}}
+      - run: yarn install --ignore-engines
+      - name: lint code
+        run: yarn lerna run lint
+      - name: run tests
+        run: yarn lerna run test --stream
+      - name: coverage
+        uses: codecov/codecov-action@v1
+        with:
+          fail_ci_if_error: true
+          verbose: false
+        env:
+          CI: true
+  publish:
+    name: publishing the module
+    runs-on: 'ubuntu-latest'
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    needs: test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10.x'
+      - name: install dependencies
+        run: yarn install --ignore-engines
```

Comment:
By using `actions/cache@v2` you can cache the `yarn cache` and next installation would be 90% faster. If is too much work I can follow up a after this one is being merged 🙃 .

Reply:
Yep, we can do that.
Could you also follow-up with adding the following test that we have in the original Travis yml which is missing in this PR?  https://github.com/lirantal/lockfile-lint/pull/110/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L22-L26

If so, I'll merge this one in, and wait for your next one to fix the cache and add that test_locales step too.

Reply:
Sure, the `test_locales`. I'll take care of it.

Reply:
@juan ready for you since this is now merged in :-)

Reply:
:) working on it ...

Reply:
That was fast, thank you! :-)

Reply:
@juan :D also can contribute if you like :D

Reply:
😆